docs/operator/migration.md

```diff
@@ -109,22 +109,22 @@ kubectl apply -f <path_to_your_wavefront.yaml>
 #### References:
```

Comment:
### 3. Re-apply your `wavefront.yaml` --> ### 3. Reapply Your `wavefront.yaml` File
### 4. Verify The Operator Status -- > ### 4. Verify the Operator Status
### 5. Check That You Are Receiving Metrics From The Operator --> ## 5. Verify That You Are Receiving Metrics from the Operator
### 7. Cleanup `wavefront.yaml` --> ### 7. Clean Up the `wavefront.yaml` File

Reply:
Hi Margarita - we don't really understand what the requested change is here.